.github/workflows/main.yml

```
@@ -0,0 +1,71 @@
+# Copyright 2019 Communication Service/Software Laboratory, National Chiao Tung University (free5gc.org)
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Main workflow
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

To make it similar to the GHA file in the other repos
```suggestion
  pull_request:
    branches:
      - main
  push:
    branches:
      - main
```

---

Done